funnel/assets/sass/update.scss

```diff
@@ -39,7 +39,7 @@
   .update__content {
     display: inline-block;
     width: 100%;
-    word-break: break-word;
+    word-break: normal;
```

Comment:
`break-word` was introduced in #813 because some URLs were so long that they were overflowing into the sidebar, and causing side scroll on mobile. [MDN docs suggest the modern replacement](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break) is:

```css
word-break: normal;
overflow-wrap: break-word;
```